app/controllers/application_controller.rb

```diff
@@ -29,8 +29,20 @@ def set_raven_context
       Raven.extra_context(params: params.to_unsafe_h, url: request.url)
     end
 
+    def sort_column
+      default_sort_column
+    end
+
     def sort_direction
-      %w[asc desc].include?(params[:direction]) ? params[:direction] : "asc"
+      %w[asc desc].include?(params[:direction]) ? params[:direction] : default_sort_direction
+    end
+
+    def default_sort_column
+      "created_at"
+    end
+
+    def default_sort_direction
+      "asc"
```

Comment:
@hengsokly _default_sort_direction_ should be **desc** to keep the latest record on the top